src/events/EventListener.ts

```diff
@@ -0,0 +1,105 @@
+import Modal, { ratingType } from "../components/layout/Modal";
+import EventBus from "./EventBus";
+import {
+  isElement,
+  isForm,
+  isHTMLElement,
+  isImage,
+  isInput,
+} from "./utils/guards";
+import { EVENT_TYPES } from "./types/eventTypes";
+
+const eventBus = EventBus.getInstance();
+
+const SELECTORS = {
+  closeModalButton: "#closeModal",
+  modalBackground: ".modal-background",
+  movieItem: ".thumbnail-list .item, .top-rated-button",
+  searchForm: ".top-rated-search",
+  searchInput: ".top-rated-search-input",
+  ratingStar: ".star",
+};
+
+export function initializeDomEventListener() {
+  window.addEventListener("click", handleClick);
+  window.addEventListener("submit", handleSubmit);
+  window.addEventListener("keydown", handleKeydown);
+}
+
+function handleClick(event: MouseEvent) {
+  const { target } = event;
+  if (!isElement(target)) return;
```

Comment:
아래처럼 작성하는 방법도 있을거 같네요

```suggestion
function handleClick({ target }: MouseEvent) {
  if (!isElement(target)) return;
```

Reply:
https://github.com/woowacourse/javascript-movie-review/pull/251/commits/3a11fa26dbab81e905472039e4f41c4f1c7af0c8 수정했습니다! 감사합니다 ㅎㅎ